janusgraph-test/src/test/java/org/janusgraph/core/ConfiguredGraphFactoryTest.java

```diff
@@ -26,16 +27,35 @@
 import org.apache.tinkerpop.gremlin.server.Settings;
 import org.apache.commons.configuration.MapConfiguration;
 
+import java.time.Duration;
 import java.util.Map;
 import java.util.HashMap;
 
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.AfterEach;
+import org.testcontainers.containers.CassandraContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import static org.janusgraph.graphdb.configuration.GraphDatabaseConfiguration.STORAGE_HOSTS;
+import static org.janusgraph.graphdb.configuration.GraphDatabaseConfiguration.STORAGE_PORT;
 import static org.junit.jupiter.api.Assertions.*;
 
+@Testcontainers
 public class ConfiguredGraphFactoryTest {
+
+    private static final String CASSANDRA_VERSION = "3.11.6";
+    private static final String CASSANDRA_IMAGE = "cassandra";
+
+    @Container
+    public static final CassandraContainer cqlContainer;
     private static final JanusGraphManager gm;
+
     static {
+        cqlContainer = new CassandraContainer<>(CASSANDRA_IMAGE + ":" + CASSANDRA_VERSION)
+            .withExposedPorts(9042)
+            .withStartupAttempts(1)
+            .withStartupTimeout(Duration.ofMinutes(10));
```

Comment:
This long delay was needed because the Cassandra docker image takes around 5 min to get ready (measured in my machine) when running the `verify` command so I choose to set a safe default. I believe it's because the tests are running in parallel.

Reply:
I think it would be better use the JanusGraphCassandraContainer.

Reply:
I did try to use JanusGraphCassandraContainer but I had issues with the tests that use "inmemory" storage. As far as I understand JanusGraphCassandraContainer replace all values on StandarStorageManager for some reason [here](https://github.com/JanusGraph/janusgraph/blob/master/janusgraph-cql/src/test/java/org/janusgraph/JanusGraphCassandraContainer.java#L178). If I comment the call to setWrapperStorageManager all tests cases works. I might be missing something 🤔

Reply:
Your assumption is correct.

Reply:
I push this change. `ConfiguredGraphFactoryTest` is now an abstract class extended by `CQLConfiguredGraphFactoryTest` and `InmemoryConfiguredGraphFactoryTest`.